memo/ts/package.json

```diff
@@ -1,18 +1,26 @@
 {
   "name": "@solana/spl-memo",
-  "version": "0.1.0",
+  "version": "0.2.0",
   "description": "SPL Memo Program JS API",
   "files": [
-    "dist",
+    "lib",
     "src"
   ],
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "main": "lib/cjs/index.js",
+  "module": "lib/esm/index.mjs",
+  "types": "lib/types/index.d.ts",
   "type": "module",
   "sideEffects": false,
+  "exports": {
+    "import": "./lib/esm/index.mjs",
+    "require": "./lib/cjs/index.js"
+  },
   "scripts": {
-    "build": "npm run clean && tsc",
-    "clean": "rimraf ./dist",
+    "build": "yarn clean && tsc -p tsconfig.json && tsc-esm -p tsconfig.json && tsc -p tsconfig.cjs.json",
+    "postbuild": "echo '{\"type\":\"commonjs\"}' > lib/cjs/package.json && echo '{\"type\":\"module\"}' > lib/esm/package.json",
+    "clean": "rimraf ./lib",
```

Comment:
`rimraf` isn't listed as a devDependency, so we should add that.

Reply:
Weird, I thought it was provided by default somewhere :sweat_smile: